code/modules/mining/mine_items.dm

```diff
@@ -278,153 +278,60 @@
 	icon_state = "capsule"
 	icon = 'icons/obj/mining.dmi'
 	w_class = 1
+	origin_tech = "engineering=3;bluespace=3"
+	var/template_id = "shelter_alpha"
+	var/datum/map_template/shelter/template
 	var/used = FALSE
 
+/obj/item/weapon/survivalcapsule/proc/get_template()
+	if(template)
+		return
+	template = shelter_templates[template_id]
+	if(!template)
+		throw EXCEPTION("Shelter template ([template_id]) not found!")
+		qdel(src)
```

Comment:
Yup, will be changing the error proc--that said, it's not really necessary to return 1 or 0 in this case. If the template fails to load, something has gone *horrifically* wrong already; it's like a human mob that ends up not having a species; it's really not possible, and on the rare instance it is, it would be better if it did runtime.